hyperspy/tests/drawing/test_plot_markers.py

```diff
@@ -472,4 +472,42 @@ def test_plot_eds_lines():
     s.plot(True)
     s.axes_manager.navigation_axes[0].index = 1
     return s._plot.signal_plot.figure
-    
+
+@update_close_figure
```

Comment:
Why do you need this?

Reply:
I guess you refer to the `import BaseSignal`? I use a basesignal to test in `test_iterate_markerks`

Reply:
No, I meand ``@update_close_figure``

Reply:
I think that is needed to close the figure that is created when `add_marker` is called.

Reply:
On second thought, it seems it is not needed.

Reply:
I think that the purpose of this in other tests is to test if closing the plot closes the markers without issue. Having it here may make this test fail if that gets broken what may be confusing. However, @ericpre should know better.